app/src/main/res/layout/item_library.xml

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<androidx.cardview.widget.CardView
+<androidx.cardview.widget.CardView xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
   android:layout_height="wrap_content"
+  android:theme="@style/Widget.KiwixTheme.CardView"
```

Comment:
you don't need to set the theme on the individual CardViews

If you look at the androidManifest

```
<application
    android:allowBackup="true"
    android:fullBackupContent="@xml/backup_rules"
    android:hardwareAccelerated="true"
    android:label="@string/app_name"
    android:supportsRtl="true"
    android:theme="@style/KiwixTheme"
    android:usesCleartextTraffic="true"
    tools:targetApi="m">
```

The theme `android:theme="@style/KiwixTheme"` is set for the entire application (you can also set the theme on an activity in the manifest but we have chosen not to for simplicity's sake) and since themes are like hierarchal styles, this theme gets inherited by all our components in  every activity.

Because our theme has been updated with this `<item name="cardViewStyle">@style/Widget.KiwixTheme.CardView</item>` what that says in English is "When using this theme automatically apply this style to all CardViews"

Also you raised a great point in a comment I can no longer find
>Should we be using CardView or MaterialCardView?

It is always difficult to know because of the custom inflater offered by the material library that replaces components eg
>Note: <Button> is auto-inflated as <com.google.android.material.button.MaterialButton> via MaterialComponentsViewInflater when using a non-Bridge Theme.MaterialComponents.* theme.

After consulting the [material.io site](https://material.io/develop/android/components/cards/), which should be considered the android bible of design, it seems like we should absolutely be using MaterialCardView. It in fact extends CardView and gives us extra features

Reply:
Great explanation @macgills I'll remove the theme from indiviual CardView ,use material cardView and update the commit accordingly.